app/models/log.rb

```diff
@@ -343,31 +343,27 @@ def reset_invalidated_dependent_fields!
   PIO = PostcodeService.new
 
   LA_CHANGES = {
-    "E07000027" => "E06000064", # Barrow-in-Furness => Westmorland and Furness
-    "E07000030" => "E06000064", # Eden => Westmorland and Furness
-    "E07000031" => "E06000064", # South Lakeland => Westmorland and Furness
-    "E07000026" => "E06000063", # Allerdale => Cumberland
-    "E07000028" => "E06000063", # Carlisle => Cumberland
-    "E07000029" => "E06000063", # Copeland => Cumberland
-    "E07000163" => "E06000065", # Craven => North Yorkshire
-    "E07000164" => "E06000065", # Hambleton => North Yorkshire
-    "E07000165" => "E06000065", # Harrogate => North Yorkshire
-    "E07000166" => "E06000065", # Richmondshire => North Yorkshire
-    "E07000167" => "E06000065", # Ryedale => North Yorkshire
-    "E07000168" => "E06000065", # Scarborough => North Yorkshire
-    "E07000169" => "E06000065", # Selby => North Yorkshire
-    "E07000187" => "E06000066", # Mendip => Somerset
-    "E07000188" => "E06000066", # Sedgemoor => Somerset
-    "E07000246" => "E06000066", # Somerset West and Taunton => Somerset
-    "E07000189" => "E06000066", # South Somerset => Somerset
```

Comment:
These would no longer get triggered, cause they're from 2023 and the APIs would now be up to date